testing/gearmand/gearmand.pre-install

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+## add gearmand group or use exsiting
+if ! getent group gearmand &>/dev/null; then
```

Comment:
Syntax `&>` is a bash-ism. Busybox ash supports it, so it’ll work on Alpine, but we prefer to avoid them for better compatibility, if there’s no a good reason to do the opposite.

Also there’s no need to check if the user/group exists, because adduser/addgroup already does this for you and you’re redirecting their stdout/stderr to /dev/null anyway.

So you can just write the following and it’ll behave exactly the same.

```#!/bin/sh

addgroup -S gearmand >/dev/null 2>&1
adduser -H -S -G gearmand -h /var/lib/gearmand -s /bin/false \
	-g "Gearman Server" gearmand >/dev/null 2>&1

exit 0
```